src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -63,7 +63,7 @@ public function exists($name)
      *
      * @throws \Twig_Error_Loader if the template could not be found
      */
-    protected function findTemplate($template)
+    protected function findTemplate($template, $throw = true)
     {
         $logicalName = (string) $template;
```

Comment:
Indeed, it would fix the tests, but that breaks the general use case. The problem here is that the loader can be a Symfony one (expecting an object) and/or a Twig one (expecting a string).